lib/services/project-v4-migration-service.ts

```diff
@@ -0,0 +1,60 @@
+import * as path from "path";
+import * as shell from "shelljs";
+
+export class AndroidResourcesV4MigrationService implements IAndroidResourcesV4MigrationService {
+	private static ANDROID_DIR = "Android";
+	private static ANDROID_DIR_TEMP = "Android-Updated";
+	private static ANDROID_DIR_OLD = "Android-Pre-v4";
+
+	constructor(private $fs: IFileSystem,
+		private $logger: ILogger,
+		private $devicePlatformsConstants: Mobile.IDevicePlatformsConstants) { }
+
+	canMigrate(platformString: string): boolean {
+		if (platformString.toLowerCase() === this.$devicePlatformsConstants.iOS.toLowerCase()) {
```

Comment:
just return `platformString.toLowerCase() !== this.$devicePlatformsConstants.iOS.toLowerCase()`

However, I believe we want to return true only in case the platformString is Android, so maybe you should return:
```TypeScript
public canMigrate(platformString: string): boolean {
    return platformString.toLowerCase() === this.$devicePlatformConstants.Android.toLowerCase();
}
```